submissions/askuznetsov/html-css-popup/style.scss

```diff
@@ -0,0 +1,229 @@
+$color-main: #d0d0ce;
+$color-secondary: royalblue;
+$border-main: 2px solid $color-secondary;
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  background-color: $color-main;
+  box-sizing: border-box;
+}
+
+.header {
+  margin: 24px 24px;
+
+  &__nav {
+    display: block;
+    padding: 8px;
+
+    &__list {
+      list-style-type: none;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      flex-wrap: wrap;
+
+      &__item {
+        height: 24px;
+        margin: 24px;
+
+        &:hover {
```

Comment:
You can get rid of this `hover` as you use links inside this list item

Reply:
Changed.